src/openrct2/interface/InteractiveConsole.cpp

```diff
@@ -1518,19 +1518,17 @@ static int32_t cc_mp_desync(InteractiveConsole& console, const arguments_t& argv
         desyncType = atoi(argv[0].c_str());
     }
 
-    std::vector<rct_sprite*> peeps;
-    std::vector<rct_sprite*> vehicles;
+    std::vector<Peep*> peeps;
 
     for (int i = 0; i < MAX_SPRITES; i++)
     {
         rct_sprite* sprite = get_sprite(i);
         if (sprite->generic.sprite_identifier == SPRITE_IDENTIFIER_NULL)
             continue;
 
-        if (sprite->generic.sprite_identifier == SPRITE_IDENTIFIER_PEEP)
-            peeps.push_back(sprite);
-        else if (sprite->generic.sprite_identifier == SPRITE_IDENTIFIER_VEHICLE)
-            vehicles.push_back(sprite);
```

Comment:
Why is the `vehicles` vector gone?

Reply:
It wasn't used for anything so thought there was no harm removing it. This function went through a few different versions during testing so there was a bit of dead code.